src/components/landing/landing-page.tsx

```
@@ -0,0 +1,494 @@
+								<Text className="text-muted-foreground leading-relaxed">
+									{faq.answer}
+								</Text>
+							</AccordionContent>
+						</AccordionItem>
+					))}
+				</Accordion>
+
+				{/* CTA at bottom */}
+				<View className="text-center mt-16">
+					<Text className="text-muted-foreground mb-6">
+						Still have questions? We're here to help.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)